BootloaderCorePkg/BootloaderCorePkg.dec

```diff
@@ -112,6 +112,18 @@
 
   gPlatformModuleTokenSpaceGuid.PcdSmbiosTablesSize       |     0x1000 | UINT16 | 0x200000E3
 
+  gPlatformModuleTokenSpaceGuid.PcdIoApicCount            |          1 | UINT32 | 0x200000E4
+  gPlatformModuleTokenSpaceGuid.PcdIntrSrcOverrideCount   |          2 | UINT32 | 0x200000E5
+  gPlatformModuleTokenSpaceGuid.PcdIntrSrcSciTrigger      |          1 | UINT16 | 0x200000E6
+  gPlatformModuleTokenSpaceGuid.PcdIntrSrcSciPolarity     |          1 | UINT16 | 0x200000E7
+  gPlatformModuleTokenSpaceGuid.PcdLocalApicNmiCount      |          1 | UINT32 | 0x200000E8
+  gPlatformModuleTokenSpaceGuid.PcdLocalApicNmiTrigger    |          1 | UINT16 | 0x200000E9
+  gPlatformModuleTokenSpaceGuid.PcdLocalApicNmiPolarity   |          0 | UINT16 | 0x200000EA
+  gPlatformModuleTokenSpaceGuid.PcdLocalApicNmiLintIdx    |          1 | UINT8  | 0x200000EB
+  gPlatformModuleTokenSpaceGuid.PcdLocalApicAddress       | 0xFEE00000 | UINT32 | 0x200000EC
+  gPlatformModuleTokenSpaceGuid.PcdIoApicAddress          | 0xFEC00000 | UINT32 | 0x200000ED
+  gPlatformModuleTokenSpaceGuid.PcdIoApicId               |          1 | UINT8  | 0x200000EE
+

```

Comment:
It would be great if adding description for these new definitions.

Reply:
There is some description in BuildLoader.py. Let me add more at the same place.

Reply:
Will all of these PCDs have to change for each project? Seems like quite a bit of configuration. If some of them are constant, then we may want to reduce these.

Reply:
BuildLoader.py is just one of the consumer to this PCD. DEC file is the place to have full description by EDK2.

Reply:
Mostly not changed. Current Madt.h of each platforms also has fixed value, but build time configurable.

Some PCD are Silicon specific. For example,
- QEMU/APL
  PcdIntrSrcSciPolarity: Active Low
  PcdLocalApicNmiCount: 1
  PcdLocalApicNmiTrigger: Level Trigger
- WHL/CFL
  PcdIntrSrcSciPolarity: Active High
  PcdLocalApicNmiCount: 16
  PcdLocalApicNmiTrigger: Edge Trigger

Other platforms may also have these changes only.

Reply:
Okay, let me put description in DEC file.